collie/model/base/base_pipeline.py

```diff
@@ -815,23 +818,21 @@ def user_user_similarity(self, user_id: int) -> pd.Series:
         user_embeddings = self._get_user_embeddings()
         user_embeddings = user_embeddings / user_embeddings.norm(dim=1)[:, None]
 
-        sim_score_idxs_series = self._similarity(embeddings=user_embeddings, id=user_id)
+        sim_score_idxs_series = self._calculate_embedding_similarity(
+            embeddings=user_embeddings,
+            id=user_id
+        )
 
         return sim_score_idxs_series
 
-    def _similarity(self, embeddings, id: int) -> pd.Series:
-        sim_score_idxs = (
+    def _calculate_embedding_similarity(self, embeddings: torch.tensor, id: int) -> pd.Series:
```

Comment:
Can you please add a little docstring here? You don't need the `Parameters` section and all that, just a line or two describing what this helper method does.